lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -116,11 +116,20 @@
             },
 
             render: function () {
+                var linkTitle;
+                if (this.options.connected) {
+                    linkTitle = gettext('Unlink');
+                } else if (this.options.acceptsLogins) {
+                    linkTitle = gettext('Link')
+                } else {
+                    linkTitle = ''
+                }
```

Comment:
Shouldn't a provider that isn't accepting logins and isn't connected just not have a link at all, rather than just having an empty title? I feel like I'm missing something in the context of this function.